src/lib/apollo/client.ts

```diff
@@ -16,21 +25,38 @@ if (typeof window !== 'undefined' && process.env.NODE_ENV === 'production') {
   });
 }
 
-const makeClient = (apiToken: string) => {
+const makeClient = (apiToken: string, routerReplace: Router['replace']) => {
   const client = new ApolloClient({
     link: from([
       makeAuthLink(apiToken),
       onError(({ graphQLErrors, networkError }) => {
         // Don't show sign out and display errors on the login page because the user won't be logged in
         if (graphQLErrors && window.location.pathname !== '/login') {
-          graphQLErrors.forEach(({ message, extensions }) => {
-            if (extensions?.code === 'AUTHENTICATION_ERROR') {
+          graphQLErrors.forEach((graphQLError) => {
+            if (graphQLError.extensions.code === 'AUTHENTICATION_ERROR') {
               signOut({ redirect: true, callbackUrl: 'signOut' }).then(() => {
                 clearDataDogUser();
                 client.clearStore();
               });
             }
-            snackNotifications.error(message);
+            if (isAccountListNotFoundError(graphQLError)) {
+              client
+                .query<GetDefaultAccountQuery>({
+                  query: GetDefaultAccountDocument,
+                })
+                .then((response) => {
+                  // eslint-disable-next-line no-console
+                  console.log('Incorrect accountListId provided. Redirecting.');
```

Comment:
I assume we want to keep this console log?

Reply:
Yes we are hoping to see this console log in DataDog so we can see how often people click links that include invalid accountListId's